src/Integration.Vsix/Commands/HelpMenu/AboutDialog.xaml

```diff
@@ -0,0 +1,39 @@
+﻿<platformUi:DialogWindow x:Class="SonarLint.VisualStudio.Integration.Vsix.Commands.AboutDialog"
```

Comment:
Fonts and colors used are from -> https://zeroheight.com/3bcff7bd1/p/64e669-sonar-brand-guide